WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Navigation/MigrationFlowCoordinator.swift

```diff
@@ -1,20 +1,30 @@
 import Combine
+import UserNotifications
 
 /// Coordinator for the migration to jetpack flow
 final class MigrationFlowCoordinator: ObservableObject {
 
     @Published private(set) var currentStep = MigrationStep.welcome
```

Comment:
Perhaps we should add a documentation here mentioning that `currentStep` values are published in a background thread.

I'm pretty sure future developers can easily know this by reading the `transitionToNextStep` implementation details but it would be great to make finding about this detail extra easier.

Reply:
done!